.github/workflows/wda-package.yml

```diff
@@ -57,3 +62,118 @@ jobs:
         uses: actions/upload-artifact@v3.1.0
         with:
           path: "${{ env.ZIP_PKG_NAME_TVOS }}"
+
+
+  for_simulator_devices_x86_64:
+    name: Build WDA for simulator iOS and tvOS devices - x86-64
+    runs-on: macos-12
```

Comment:
it is better to move macos version to a variable as xcode version depends on it

Reply:
do you know what we could use? `env` key as same as `XCODE_VERSION` got an error for the runs-on..

Reply:
maybe we can try https://stackoverflow.com/questions/71961921/specify-runner-to-be-used-depending-on-condition-in-a-github-actions-workflow